tests/functionaltests/FunctionalTests.cpp

```diff
@@ -691,7 +697,7 @@ class ProjectionTest {
     }
 
     TEST_METHOD_CLEANUP(ProjectionTestCleanup) {
-        FunctionalTestCleanupUIApplication();
+        (void)FrameworkHelper::RunOnUIThread(&FunctionalTestCleanupUIApplication);
```

Comment:
also - if RunOnUIThread returns an HRESULT, we should return SUCCEEDED(FrameworkHelper::RunOnUIThread(...

---
In reply to: [94668773](https://github.com/Microsoft/WinObjC/pull/1627#discussion_r94668773) [](ancestors = 94668773,94668735)